docs/installation_guide.md

```diff
@@ -1,292 +1,316 @@
+
+
+### Managing Harbor's lifecycle
+You can use docker-compose to manage the lifecycle of Harbor. Some useful commands are listed as follows (must run in the same directory as *docker-compose.yml*).
+
+Stopping Harbor:
+```
+$ sudo docker-compose stop
+Stopping nginx ... done
+Stopping harbor-jobservice ... done
+Stopping harbor-ui ... done
+Stopping harbor-db ... done
+Stopping registry ... done
+Stopping harbor-log ... done
+```  
+Restarting Harbor after stopping:
+```
+$ sudo docker-compose start
+Starting log ... done
+Starting ui ... done
+Starting mysql ... done
+Starting jobservice ... done
+Starting registry ... done
+Starting proxy ... done
+```  
+
+To change Harbor's configuration, first stop existing Harbor instance, update harbor.cfg, and then run install.sh again:
+```
+$ sudo docker-compose down -v
+
+$ vim harbor.cfg
+
+$ sudo install.sh
+``` 
+
+Removing Harbor's containers while keeping the image data and Harbor's database files on the file system:
+```
+$ sudo docker-compose down -v
+```  
+
+Removing Harbor's database and image data (for a clean re-installation):
+```sh
+$ rm -r /data/database
+$ rm -r /data/registry
+```
+
+**Note** When Harbor is installed with Notary, user needs to add extra template file to docker-compose command, so the docker-compose commands to manage the lifecycle of Harbor will be:
+```
+sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.notary.yml [ up|down|stop|start ]
+```
+
+Please check the [Docker Compose command-line reference](https://docs.docker.com/compose/reference/) for more on docker-compose.
+
+### Persistent data and log files
+By default, registry data is persisted in the target host's `/data/` directory.  This data remains unchanged even when Harbor's containers are removed and/or recreated.  
+
+In addition, Harbor uses *rsyslog* to collect the logs of each container. By default, these log files are stored in the directory `/var/log/harbor/` on the target host for troubleshooting.  
+
+## Configuring Harbor listening on a customized port
+By default, Harbor listens on port 80(HTTP) and 443(HTTPS, if configured) for both admin portal and docker commands, you can configure it with a customized one.  
+
+### For HTTP protocol
+
+1.Modify docker-compose.yml  
+Replace the first "80" to a customized port, e.g. 8888:80.  
+
+```
+proxy:
+    image: library/nginx:1.11.5
+    restart: always
+    volumes:
+      - ./config/nginx:/etc/nginx
+    ports:
+      - 8888:80
+      - 443:443
+    depends_on:
+      - mysql
+      - registry
+      - ui
+      - log
+    logging:
+      driver: "syslog"
+      options:  
+        syslog-address: "tcp://127.0.0.1:1514"
+        tag: "proxy"
+```
+
+2.Modify templates/registry/config.yml  
```

Comment:
Modifying hostname in harbor.cfg by adding the port has the same effect and can also make the docker pull command to be shown correctly on UI.